### PR TITLE
esmf: update home page URL in Portfile

### PR DESCRIPTION
**Pull request by Dave-Allured**
Update ESMF home page URL in portfile.
* Portfile change only.
* Did not change or look for any internal URL's in code or docs.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

(Not applicable)

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] checked your Portfile with `port lint`?

### CODE REVIEW

File: science/esmf/Portfile [hunk 1]
```diff
@@ -22,7 +22,7 @@ description         software for building and coupling weather, climate, and rel
 long_description    The ESMF defines an architecture for composing complex, coupled \
                     modeling systems and includes data structures \
                     and utilities for developing individual models.
-homepage            https://www.earthsystemcog.org/projects/esmf/
+homepage            http://www.earthsystemmodeling.org
```
Comment (reneeotten) [inline science/esmf/Portfile:25]: please use `https`

Comment (Dave-Allured) [reply science/esmf/Portfile:25]: That website currently supports only http, not https.  Please keep this URLas http, until such time as the website is updated for https.

Comment (reneeotten) [reply science/esmf/Portfile:25]: this works fine for me: https://earthsystemmodeling.org/

Comment (Dave-Allured) [reply science/esmf/Portfile]: Okay, thank you for finding that.  Sorry I missed it the first time.  I pushed the new URL as you suggested.  Is this okay now?

**Pull request merged**